shared/teams/add-members-wizard/confirm.tsx

```diff
@@ -324,6 +312,8 @@ const AddingMember = (props: Types.AddingMember & {lastMember?: boolean}) => {
         {showDropdown && (
           <FloatingRolePicker
             open={showingMenu}
+            presetRole={individualRole}
+            onCancel={individualRole === rolePickerRole ? () => setShowingMenu(false) : undefined}
```

Comment:
:sheep: would it be much harder to close the menu regardless? feels weird to me for it to close sometimes

Reply:
Hmm. I think a lot of places misuse the props of the role picker and change their internal state as soon as a new role is selected, so cancelling that seems ill-defined.